### PR TITLE
Add loading spinner for insight widgets while they're in a loading state

### PR DESCRIPTION
**Pull request by kevcunnane**
- This was very confusing without status indicator, is it complete or in progress?

Please let me know if you see issues with this e.g. was there a reason we didn't do this up to now?

Initial state:

image

On loading / error:
image

### CODE REVIEW

**Comment by kevcunnane** [comment]
@anthonydresser can you review? I didn't get any response on original PR

File: src/vs/monaco.d.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ declare namespace monaco {
 	}
 
 
-	export const enum MarkerTag {
+	export enum MarkerTag {
```
Comment (anthonydresser) [inline src/vs/monaco.d.ts]: Something tells me this isn't meant to be here.

Comment (anthonydresser) [reply src/vs/monaco.d.ts]: You might needs to check out your changes to this file, clean and recompile

**Comment by kevcunnane** [comment]
@Anthony I think this showed due to the accidental pushes to Master Raj did. If you look now there are only 2 files showing changed

**Pull request merged**